parser-typechecker/src/Unison/Runtime/ANF/Serialize.hs

```diff
@@ -0,0 +1,560 @@
+{-# language LambdaCase #-}
+{-# language BangPatterns #-}
+
+module Unison.Runtime.ANF.Serialize where
```

Comment:
This file must have been fun to write. :)

Reply:
Super fun.